fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/AnnouncementCardTokens.kt

```diff
@@ -0,0 +1,115 @@
+package com.microsoft.fluentui.theme.token.controlTokens
+
+import android.os.Parcelable
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.AliasTokens
+import com.microsoft.fluentui.theme.token.GlobalTokens
+import com.microsoft.fluentui.theme.token.StateColor
+import kotlinx.parcelize.Parcelize
+
+data class AnnouncementCardInfo(
+    override val cardType: CardType = CardType.Elevated,
+    val buttonType: ButtonStyle = ButtonStyle.TextButton
+) : BasicCardControlInfo()
+
+@Parcelize
+open class AnnouncementCardTokens : BasicCardTokens(), Parcelable {
+    @Composable
+    open fun textColor(announcementCardInfo: BasicCardControlInfo): Color {
+        return FluentTheme.aliasTokens.neutralForegroundColor[AliasTokens.NeutralForegroundColorTokens.Foreground2].value(
+            themeMode = FluentTheme.themeMode
+        )
+    }
+
+    @Composable
+    open fun titleColor(announcementCardInfo: BasicCardControlInfo): Color {
+        return FluentTheme.aliasTokens.neutralForegroundColor[AliasTokens.NeutralForegroundColorTokens.Foreground1].value(
+            themeMode = FluentTheme.themeMode
+        )
+    }
+
+    @Composable
+    open fun titleSize(announcementCardInfo: BasicCardControlInfo): TextStyle {
+        return FluentTheme.aliasTokens.typography[AliasTokens.TypographyTokens.Body1Strong]
+    }
+
+    @Composable
+    open fun textSize(announcementCardInfo: BasicCardControlInfo): TextStyle {
```

Comment:
Can we rename this to typography with appropriate prefix. Would be coherent all across.